sessionctx/variable/session.go

```diff
@@ -46,7 +46,7 @@ import (
 	"github.com/pingcap/tidb/types"
 	"github.com/pingcap/tidb/util/chunk"
 	"github.com/pingcap/tidb/util/execdetails"
-	utilMath "github.com/pingcap/tidb/util/math"
+	utilMath "github.com/pingcap/tidb/util/mathutil"
```

Comment:
Please remove the alias and update the file to use the mathutil namespace instead.